make-nagios-commands-cfg.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import os
+
+from jinja2 import Environment, FileSystemLoader, Template
+
+smtp_server = os.getenv('SMTP_SERVER', 'localhost')
```

Comment:
The change would be:

```diff
-smtp_server = os.getenv('SMTP_SERVER', 'localhost')
+smtp_server = os.environ['SMTP_SERVER']
```

Accessing a key that doesn't exist will raise an exception. If we wanted to be fancy:

```python
import sys
# ...

try:
    smtp_server = os.environ['SMTP_SERVER']
    # ...
except KeyError:
    print "Missing required environment variable"
    sys.exit(1)
```

For our usage with the template, we'll always be setting those environment variables anyway...